modules/library/main/src/main/java/org/geotools/filter/function/FilterFunction_equalTo.java

```diff
@@ -31,9 +31,10 @@
  */
 public class FilterFunction_equalTo extends FunctionExpressionImpl {
     
-    public static FunctionName NAME = new FunctionNameImpl("equalTo", Boolean.class,
-            parameter("object1", Object.class),
-            parameter("object2", Object.class));
+    public static FunctionName NAME = new FunctionNameImpl("equalTo", 
+    		parameter("equalTo", Boolean.class,"equal to","Can be used to compare for equality two numbers, two strings, two dates, and so on."),
```

Comment:
Looks like we did not set up your IDE code-formatting properly; this tab vs space thing messes us up (see the developers guide).